perun-ldapc/src/main/resources/perun-ldapc.xml

```diff
@@ -598,6 +622,34 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 		</property>
 	</bean>
 
+	<bean id="perunFacility" class="cz.metacentrum.perun.ldapc.model.impl.PerunFacilityImpl">
+		<property name="attributeDescriptions">
+			<!--  this list will be added to built-in defaultAttributeDescriptions -->
+			<list>
```

Comment:
Here is the list of attributes, which we need in LDAP by default :

urn:perun:facility:attribute-def:def:OIDCClientID
urn:perun:facility:attribute-def:def:CheckGroupMembership
urn:perun:facility:attribute-def:virt:voShortNames
urn:perun:facility:attribute-def:def:wayfFilter
urn:perun:facility:attribute-def:def:wayfEFilter
urn:perun:facility:attribute-def:def:entityID
urn:perun:facility:attribute-def:def:isTestSp
urn:perun:facility:attribute-def:def:proxyIdentifiers
urn:perun:facility:attribute-def:def:showOnServiceList

Reply:
Just a note - for ``facility:virt:voShortNames`` we need to update attribute module in order to generate messages about changes.

Reply:
Please write the requirements to the respective Redmine issues (there are two, AFAIK - one specifically for facility class use-cases, one requesting the implementation).